Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -3720,6 +3720,27 @@ LayoutUnit RenderBox::constrainBlockMarginInAvailableSpaceOrTrim(const RenderBox
     return marginSide == MarginTrimType::BlockStart ? minimumValueForLength(style().marginBeforeUsing(&containingBlock.style()), availableSpace) : minimumValueForLength(style().marginAfterUsing(&containingBlock.style()), availableSpace);
 }
 
+bool RenderBox::hasTrimmedMargin(MarginTrimType marginTrimType) const
+{
+    ASSERT(!needsLayout());
+    auto* container = containingBlock();
```

Comment:
I think you should stick to containingBlock name as "container" is a somewhat broader type (e.g. it could be a inline box)

Reply:
Changed